docs/scripts/deploy_docs.sh

```diff
@@ -9,7 +9,15 @@ python docs/autogen.py
 # when pydocmd stops 'serving'.
 # To get around this we:
 
+
 #   1. serve the docs locally
+
+# Kill the any process running on port 8000 if it exists, this is the port that pydocmd serves on.
+process_on_our_port=$(sudo lsof -t -i:8000)
+if [[ $process_on_our_port ]]; then
+  sudo kill $process_on_our_port
+fi
+
```

Comment:
This is fine but you can also do it in one line:

process_on_our_port=$(sudo lsof -t -i:8000) && sudo kill $process_on_our_port

Reply:
Nice - I'll add that